### DISCUSSION TITLE
Get afocus and azoom

### DISCUSSION THREAD

**Discussion by GogaView** [AxisCommunications/discussions#153] [VAPIX] [answered]
Hello!
I use /axis-cgi/opticssetup.cgi?afocus=N and /axis-cgi/opticssetup.cgi?azoom=N to set parameters. 
How to get current values of azoom and afocus?

**Comment by onie5** [comment]
Hi,
The endpoint does not show up in the VAPIX library. From where you got this URL?

**Reply by GogaView** [reply]
In a direct response from AXIS support

**Reply by onie5** [reply]
Hi,
The endpoint is for calibrating the optics, azoom=1 puts zoom to Tele while azoom=0 sets it to Wide. Likewise, afocus=1 sets focus to far, afocus=0 sets it to near.

**Reply by GogaView** [reply, marked as answer]
Hi,
I understand it, but I don't understand how to GET values of  focus to far and zoom. 
For example:
to SET Brightness I use /axis-cgi/param.cgi?action=update&ImageSource.I0.Sensor.Brightness=50;
to GET Brightness I use /axis-cgi/param.cgi?action=list&group=ImageSource.I0.Sensor.Brightness

So, How to GET afocus and azoom?
